sources/editor/Xenko.GameStudio/Xenko.GameStudio.csproj

```diff
@@ -17,13 +17,13 @@
     <XenkoAssemblyProcessorOptions>--auto-module-initializer</XenkoAssemblyProcessorOptions>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
I'm a bit confused here. Don't we support the GameStudio on both 32bits and 64bits? cc @xen2 
Also what does it fix here, since "AnyCPU" includes x64 and we also set "Prefer32Bit" to `false`.

Reply:
The warning is raised because at least one of the dependencies of these executable projects is a C++ projects that built for x64, in which case the executable project is expected to do the same.

Reply:
I wasn't sure if the C++ projects are also built for x86 and I couldn't see it in the solution build configurations at least, but if they are then first I would ask if it makes sense to cut them loose and only target one of x86/x64 to simplify things, otherwise the fix would be to also suppress the warnings for the executable projects.

Reply:
@Kryptos-FR editor has been x64 only for quite a while

Reply:
@xen2 Ah ok ^^ I wasn't sure about that.